libs/platform/src/lib/components/list/list.component.ts

```diff
@@ -528,40 +528,50 @@ export class ListComponent extends CollectionBaseInput implements OnInit, AfterV
             });
     }
 
+
     /**@hidden
     *  used in tempate to get Selected items from a list
     * event:any to avoid code duplication**/
-    _onSelectionChanged(event: any): void {
-        if (event.target.checked) {
-            this._selectionModel.select(event.target.parentNode.parentNode.parentNode);
-        } else {
-            this._selectionModel.deselect(event.target.parentNode.parentNode.parentNode);
-        }
+    _onSelectionChanged(event: Event): void {
+        this._updateNavigation(event);
     }
 
     /** @hidden */
     /**  on Update navgiation styles for non navigated items
      * event:any to avoid code duplication
      * seprate PR for custom event**/
     @HostListener('click', ['$event'])
-    _updateNavigation(event: any): void {
+    _updateNavigation(event: Event): void {
```

Comment:
`Event` => `MouseEvent`

Reply:
this function is called on mouse click as well as on touch as well.